nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -468,11 +486,9 @@ in {
           script = ''
             chmod og+x ${cfg.home}
             ln -sf ${cfg.package}/apps ${cfg.home}/
-            mkdir -p ${cfg.home}/config ${cfg.home}/data ${cfg.home}/store-apps
+            install -o ${cfg.user} -g ${group} -d ${cfg.home}/config ${cfg.home}/data ${cfg.home}/store-apps
```

Comment:
This script really shouldn't even be run as `root`, so if there are any changes here maybe it should be to fix that...